cpg-neo4j/build.gradle.kts

```diff
@@ -54,4 +54,6 @@ dependencies {
     // Command line interface support
     api(libs.picocli)
     annotationProcessor(libs.picocli.codegen)
+
+    testImplementation(testFixtures(projects.cpgCore))
```

Comment:
What is the effect of this change?

Reply:
That I can use the test utility classes in the neo4j tests. This could probably be moved into the common buildSrc convention because it makes sense for all projects.

Reply:
Ok, I see. Fine for me.